src/Contracts/Models/Evaluator.cs

```diff
@@ -1,19 +1,22 @@
-﻿using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class Evaluator
     {
-        [JsonProperty(PropertyName = "correlation_id")]
+        [ConfigurationKeyName("correlation_id")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
changed